cypress/pageobjects/ManualAggregateSyncPageObject.js

```diff
@@ -0,0 +1,117 @@
+import { dataTest } from "../support/utils";
```

Comment:
Minor: not sure about adding a new folder to cypress. Maybe using `cypress/suport`?

Reply:
Yes, no problem.

Reply:
Done!